README.md

```diff
@@ -28,6 +28,17 @@ If you like to join developing,
 Specification
 -------------
 
+### Technologies Used
+
+Badgeyay uses a number of open source projects:
+
+* [Flask](http://flask.pocoo.org/) - Microframework powered by python
+* [Bootstrap](http://getbootstrap.com/) - Responsive frontend framework
```

Comment:
We are using Bootstrap v3 so i think we should redirect to https://getbootstrap.com/docs/3.3/